src/redux/coinlore/CoinSlice.js

```diff
@@ -14,7 +14,7 @@ export const getCoins = createAsyncThunk('crypto/fetchCrypto', async (_, api) =>
     const res = await axios.get(getCoinUrl);
     return res.data;
   } catch (error) {
-    return api.rejectWithValue('Api failed to fetch');
+    return api.rejectWithValue('Api failed to fetch data');
```

Comment:
- [ ] As requested by the previous reviewer in this [comment](https://github.com/Simpleshaikh1/Metric-App/pull/5#discussion_r1320547407) on the first [Pr](https://github.com/Simpleshaikh1/Metric-App/pull/5), kindly ensure your design follows the given design

Your design
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/5a00e78e-c4a8-4044-ae71-c17480d8980b)

Given design
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/6189da6a-3804-4600-88ba-005cdfa07135)

- Kindly use the given font family for all the text in your app
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/94b0dc34-bca6-466f-b4b9-c3484aefcec2)

- The layout should also be followed, make sure there is no space around the cards, they should span the whole view. Also text should be aligned to the bottom left as in the given design

Your cards
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/67ac563a-7ca5-4693-920c-854326bc2fac)

Given cards
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/1e419fbd-2a2d-4fd2-9fce-bccdc2f2baf8)